graffiti_monkey/cli.py

```diff
@@ -24,6 +24,7 @@
 
 
 __all__ = ('run', )
+logging.basicConfig(datefmt="%Y-%m-%d %H:%M:%S")
```

Comment:
I am happy to merge this PR once this line is removed. Can you please do that @shravanpn7?